pimcore/models/Object/Classificationstore/GroupConfig/Dao.php

```diff
@@ -70,8 +70,13 @@ public function getByName($name = null)
         }
     }
 
+    /**
+     * @return int
+     */
     public function hasChildren()
     {
+        $amount = 0;
```

Comment:
Although this should be OK, please do not add something like this in this commit, as it potentially changes behaviour (calls relying on hasChildren returning a `null` or an int). Rather add a TODO, so we can fix this in a later change.